### PR TITLE
Update Main.java

### PR DESCRIPTION
**Pull request by CaptainInPHW**

### CODE REVIEW

File: src/main/java/com/github/hcsp/controlflow/Main.java [hunk 1]
```diff
@@ -9,7 +9,13 @@
      * @param start 区间开始
      * @param end 区间结束
      */
-    public static void printOddNumbersBetween(int start, int end) {}
+    public static void printOddNumbersBetween(int start, int end) {
+        for (int i = start; i <= end; i++) {
+            if (i % 2 != 0) {
+                System.out.println(i);
```
Comment (blindpirate) [inline src/main/java/com/github/hcsp/controlflow/Main.java]: 不是说了用逗号分割么。。。

Comment (CaptainInPHW) [reply src/main/java/com/github/hcsp/controlflow/Main.java]: sorry，编程不需要视力 😅

**Comment by CaptainInPHW** [comment]
Your description is like a shit，and wasted my one hour. Maybe blow is better ?
```java
public static String printOddNumbersBetween(int start, int end) {}
```
Using a string type return statement is more helpful for us to understand your purpose, do you think so?

File: src/main/java/com/github/hcsp/controlflow/Main.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
     public static void printOddNumbersBetween(int start, int end) {
         for (int i = start; i <= end; i++) {
             if (i % 2 != 0) {
-                System.out.println(i + ",");
+                System.out.print(i + ",");
```
Comment (CaptainInPHW) [inline src/main/java/com/github/hcsp/controlflow/Main.java]: WTF! But now I know the difference between `system.out.println` and `system.out.print()`

**Pull request merged**